examples/developer-tools/src/index.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as Blockly from 'blockly/core';
+import * as En from 'blockly/msg/en';
+import 'blockly/blocks';
+import './index.css';
+
+// Put Blockly in the global scope for easy debugging.
+(window as any).Blockly = Blockly;
+
+// Even though En should be loaded by default,
+// if you don't load it specifically, you'll get spurious message warnings.
```

Comment:
I think you get spurious warnings because you're loading from `blockly/core`.

Reply:
Yes, that's true. I also load the English language files separately. If I don't call `setLocale` then the default behavior is that the English language files are loaded (and everything appears to work fine if I don't call it and rely on this default behavior) but I get the warning. Calling `setLocale` changes nothing about the behavior other than silences the warnings. so I'm just explaining why I call `setLocale` even though it appears to work if you don't. our documentation used to say you only needed to do that if you didn't want english, but i noticed it does tell you now to do it in english too.